ci-operator/config/rh-messaging/qdr-shipshape/rh-messaging-qdr-shipshape-main__interconnect-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,43 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: interconnect-amp-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      OPERATORS: |
+        [
+            {"name": "amq-interconnect-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "default", "target_namespaces": "default"}
+        ]
+      SPOT_INSTANCES: "true"
```

```suggestion
      SPOT_INSTANCES: "false"
```

Your last error seemed to be an AWS error with spot instances. Lets try setting this to false and re-run to see if we can get it to pass for you

---

okay, pushed the change, lets see. Thanks!